charts/fwd-proxy/chart/values.yaml

```
@@ -42,6 +43,91 @@ nginx:
     #   cpu: 100m
     #   memory: 128Mi
 
+caddy:
+  enabled: true
+  image:
+    repository: caddy
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "2.5.2-alpine"
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  caddyfile: |
+    {
+        debug
+        auto_https disable_redirects
+        local_certs
+        skip_install_trust
+    }
+
+    :80 {
+      respond / 404
+      respond /healthz 200
+    }
+
+    {{- $registries := list "registry-1.docker.io" "ghcr.io" "k8s.gcr.io" "quay.io" "gridopsingularity.azurecr.io" }}
+    {{- range $registries }}
+    {{ . }}:443 {
+      respond / 200
+
+      reverse_proxy /v2/ http://image-registry-service
+      reverse_proxy /v2/_catalog http://image-registry-service
+
+      @pathAfterVersion {
+        path_regexp xp ^/v2/(.+)
+      }
+
+      {{- if eq . "registry-1.docker.io" }}
+      rewrite @pathAfterVersion /v2/docker.io/{re.xp.1}
+      {{- else }}
+      rewrite @pathAfterVersion /v2/{{ . }}/{re.xp.1}
+      {{- end }}
+      reverse_proxy http://image-registry-service
+
+    }
+    {{- end }}
+
+    {{- $gitRepositories := list "github.com" "dev.azure.com" }}
+    {{- range $gitRepositories }}
+    {{ . }}:443 {
+      respond / 200
+
+      reverse_proxy * http://git-server-service
+    }
+    {{- end }}
+
+    {{- $helmRepositories := list "distributed-technologies.github.io" "energinet-singularity.github.io" }}
+    {{- range $helmRepositories }}
+    {{ . }}:443 {
+
+      @withOutFolders {
+        path_regexp xp /([^/]*)$
+      }
+
+      rewrite @withOutFolders /{re.xp.1}
```

I think we can just do:
```suggestion
      rewrite * /{file}
```
`{file}` = `The filename of the path, excluding directory`, ref https://caddyserver.com/docs/caddyfile/concepts#placeholders and https://github.com/caddyserver/caddy/blob/fe61209df2d8ecbff7371703ae2dfa58342d2ed9/modules/caddyhttp/app.go#L92

---

This works fine (y)